scripts/Engineering/EnginX.py

```diff
@@ -306,13 +312,15 @@ def save_calibration(ceria_run, van_run, cal_dir, cal_gen, name, bank_names, zer
     else:
         template_file = "template_ENGINX_241391_236516_North_bank.prm"
     # write out the param file to the users directory
+
     Utils.write_ENGINX_GSAS_iparam_file(output_file=gsas_iparm_fname, bank_names=bank_names, difc=difcs, tzero=zeros,
                                         ceria_run=ceria_run, vanadium_run=van_run,
                                         template_file=template_file)
-    # copy the param file to the general directory
-    if not os.path.exists(cal_gen):
-        os.makedirs(cal_gen)
-    copy2(gsas_iparm_fname, cal_gen)
+    if not cal_gen == cal_dir:
```

Comment:
Might want to consider renaming `cal_dir` and `cal_gen` to `calibration_directory` and `calibration_general` to be consistent in the whole script